### ISSUE TITLE
Gym detail should display leader's pokemon

### ISSUE THREAD

**Issue by matejos** [mseleng/Kodemon#197]
At least in my opinion. What do you think?
(Yes you can click on leader and then see his pokemon, but that information should be readily available to you in case you want to attack gym)

**Comment by Ahimtar** [comment]
I was thinking about it, but it seems that then the gym and trainer pages are almost redundant (perhaps then the username shouldn't be shown on the gym page?) and it is a bit more clean/simple now (when you want more information about the gym's pokemon, you click on the trainer).
But if you think it's better with all information shown, I can add it.

**Comment by matejos** [comment]
I don't agree completely, for example gym detail pages are now not redundant because you can view gym's fights and also fight gym from its detail page. As long as we don't implement these features into the list, the detail pages have their purpose.
Same with the trainer page, you can view their fights and also badges (you may not really need to know gym's leader's badges when viewing gym's detail)

**Comment by mseleng** [comment]
I agree with @matejos here, because the user experience will not be good when you "force" the user to go to another page, there see information they need and do another clicks to come back and perform fight.

**Comment by Flexor28** [comment]
I also thing that user should see gym leader's pokemons on gym page to make the fight action quicker and more comfortable.
